.github/workflows/deploy.yml

```
@@ -61,22 +56,32 @@ jobs:
         if: github.ref == 'refs/heads/develop'
         uses: codfish/semantic-release-action@v1
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GH_ACTION_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
       - name: Deploy on master branch
         if: github.ref == 'refs/heads/master'
         uses: codfish/semantic-release-action@v1
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GH_ACTION_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
-      - name: Merge to master branch
-        if: github.ref == 'refs/heads/master'
+  sync_master_to_development_branch :
+    if: github.ref == 'refs/heads/master' || github.ref == 'refs/heads/develop'
+    runs-on: ubuntu-latest
+    needs: deploy
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          ref: develop
+          token: ${{ secrets.GH_ACTION_TOKEN }}
+
+      - name: Merge master into develop branch
         run: |
-          git checkout develop
+          git config --global user.email ${{ secrets.GH_EMAIL }}
+          git config --global user.name ${{ secrets.GH_USER }}
           git merge origin/master
-          git push origin develop
+          git push origin development
```

I don't think this is right, I think we are using `develop` on passport-did-auth
```suggestion
          git push origin develop
```

---

Oh yes indeed, my bad. Thank you @jrhender